### PR TITLE
5.6.7 Release notes

### PR DESCRIPTION
**Pull request by andrewvc**
`git log v5.6.6..60c90ed`

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,14 +3,32 @@
 
 This section summarizes the changes in the following releases:
 
-* <<logstash-5-6-6,Logstash 5.6.6>>
+* <<logstash-5-6-7,Logstash 5.6.6>>
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: It looks like you are linking 5-6-7 to 5-6-6 , i think you need both ?

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -3,14 +3,32 @@
...
 * <<logstash-5-6-4,Logstash 5.6.4>>
 * <<logstash-5-6-3,Logstash 5.6.3>>
 * <<logstash-5-6-2,Logstash 5.6.2>>
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-7]]
+=== Logstash 5.6.7 Release Notes
+
+*Gelf Input*
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: I think you need a blank line here for the markdown.

Comment (andrewvc) [reply docs/static/releasenotes.asciidoc]: Done

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -3,14 +3,33 @@
 
 This section summarizes the changes in the following releases:
 
-* <<logstash-5-6-6,Logstash 5.6.6>>
+* <<logstash-5-6-7,Logstash 5.6.7>>
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: Missing link to 5-6-6

**Comment by jakelandis** [comment]
LGTM

**Pull request closed without merging**